tests/test_examples.py

```diff
@@ -86,6 +100,99 @@ def test_examples_files(test_file: str, symbols_by_file: Dict[str, Set[str]]) ->
     assert found_pylint_message, f"Failed {test_file}. File does not add expected message."
 
 
+def _symbols_by_file_pyta(paths: List[str], include_msg: bool = False) -> Dict[str, Set[str]]:
+    """
+    Run python_ta on all the example files and return the map of file name to the
+    set of PythonTA messages it raises.
+
+    :param paths: The paths to retrieve the files from.
+    :param include_msg: whether to include msgs[msg] in the symbol set
+    """
+    sys.stdout = StringIO()
+    python_ta.check_all(
+        module_name=get_file_paths(paths),
+        config={
+            "output-format": "python_ta.reporters.JSONReporter",
+        }
+    )
+
+    jsons_output = sys.stdout.getvalue()
+    sys.stdout = sys.__stdout__
+    pyta_list_output = json.loads(jsons_output)
+
+    file_to_symbol = {}
+    for path, group in itertools.groupby(pyta_list_output, key=lambda d: os.path.basename(d["filename"])):
+        symbols = set()
+        for message in group:
+            for msg in message["msgs"]:
+                symbols.add(msg["symbol"])
+                if include_msg:
+                    symbols.add(msg["msg"])
+
+        file = os.path.basename(path)
+        file_to_symbol[file] = symbols
+
+    return file_to_symbol
+
+
+@pytest.fixture(scope="session")
+def check_examples_pyta() -> Dict[str, Set[str]]:
+    """
+    Checking the examples files with python_ta
+    """
+    return _symbols_by_file_pyta([_EXAMPLES_PATH, _CUSTOM_CHECKER_PATH])
+
+
+@pytest.fixture(scope="session")
+def check_pycodestyle_pyta() -> Dict[str, Set[str]]:
+    """
+    Checking the pycodestyle error files with python_ta, including the error message to
+    check for the specific error type
+    """
+    return _symbols_by_file_pyta([_PYCODESTYLE_PATH], include_msg=True)
+
+
+@pytest.mark.parametrize("test_file", get_file_paths([_EXAMPLES_PATH, _CUSTOM_CHECKER_PATH]))
+def test_examples_files_pyta(test_file: str, check_examples_pyta: Dict[str, Set[str]]) -> None:
+    """Creates all the new unit tests dynamically from the testing directory."""
```

Comment:
This docstring description and the one on line 175 are inconsistent. This is a good opportunity to improve on what came before. Make the test description start with the words "Test that" and then make clear what is being tested.